R/selections.R

```diff
@@ -221,13 +221,26 @@ recipes_eval_select <- function(quos, data, info, ..., allow_rename = FALSE,
 #' `has_role()`, `all_predictors()`, and `all_outcomes()` can be used to
```

Comment:
This file has been updated to include a lot more selector functions, now that we are able to detect them. They all have `*_predictors()` variants and have the hierarchy listed below